### ISSUE TITLE
How do we perform pre-training/fine-tuning for visual quesion answering task on custom dataset.

### ISSUE THREAD

**Issue by kirito-0512** [NVlabs/prismer#23]
I would greatly value your assistance in offering guidance for initiating pre-training/fine-tuning on the Visual Question Answering (VQA) task, specifically in the following aspects:

1. The necessary format for the required dataset.
2. Minimum hardware requirements for its execution.

Please note that while this question might be straightforward and potentially addressed by reviewing the model documentation, I am seeking an expert opinion on this matter.

Thank you sincerely.

**Comment by lorenmt** [comment]
Hello, we have released VQA checkpoints in this repo, you can try it out first to see if it works within your needs. Otherwise, you should just follow the instructions in the documentation, i.e. getting the expert labels ready and modify the training config scripts.

**Comment by kirito-0512** [comment]
Thank you!, will work on it would be higly appreciated if you could also provide any additonal resources.
